web/html/src/manager/clusters/cluster/cluster-overview.js

```diff
@@ -0,0 +1,328 @@
+    .catch((error : ErrorMessagesType) => {
+      props.setMessages(error.messages);
+    })
+  };
+
+  const onFetchClusterProps = () => {
+    setFetchingProps(true);
+    fetchClusterProps(props.cluster.id)
+      .then((data: ClusterType) => {
+        setCluster(data);
+        props.onUpdateName(data.name);
+      })
+      .catch((error : ErrorMessagesType) => {
+        props.setMessages(error.messages);
+      })      
+      .finally(() => {
+          setFetchingProps(false);
+      });    
+  }
+
+  const onSaveClusterProps = () => {
+    return saveClusterProps(props.cluster.id, editModel)
+      .then(_ => {
+        closeDialog("cluster-edit-dialog");
+        props.setMessages([Messages.success(t("Cluster properties updated successfully"))]);
+        onFetchClusterProps();
+      })
+      .catch((error : ErrorMessagesType) => {
+        props.setMessages(error.messages);
+      });
+  };
+
+  const filterFunc = (row, criteria) => {
+      const keysToSearch = ['hostname'];
+      if (criteria) {
+          return keysToSearch.map(key => row[key]).join().toLowerCase().includes(criteria.toLowerCase());
+      }
+      return true;
+  };
+
+  const onSelectNode = (event, hostname) => {
+      const newSelections = new Set(selections);
+      if (event.target.checked) {
+          newSelections.add(hostname);
+      } else {
+          newSelections.delete(hostname);
+      }
+      setSelections(newSelections);
+  }  
+
+  const editContent =
+      <Form
+          model={editModel}
+          onValidate={(valid) => console.log("valid: " + (valid ? "true" : "false"))}
+          onChange={(model) => {setEditModel(model); console.log(model)}} 
+          formDirection="form-horizontal"
+          >
+          <Text
+              name="name"
+              label={t('Name')}
+              required
+              labelClass="col-md-3"
+              divClass="col-md-6"
+              />
+          <Text
+              name="description"
+              label={t('Description')}
+              required
+              labelClass="col-md-3"
+              divClass="col-md-6"
+              />                
+      </Form>;   
+
+  const editButtons = <React.Fragment>
+          <div className="btn-group col-lg-6"></div>
+          <div className="col-lg-6">
+              <div className="pull-right btn-group">
+                  <Button
+                      id="cancel-btn"
+                      text={t("Cancel")}
+                      className="gap-right btn-default"
+                      handler={() => {
+                          closeDialog("cluster-edit-dialog");
+                      }}
+                      />
+                  <AsyncButton id="save-btn" defaultType="btn-primary"
+                      icon="fa-save"
+                      text={t("Save") }
+                      className="gap-right"
+                      action={() => onSaveClusterProps()}
+                      />
```

Comment:
As mentioned; this should be disabled with an `isValid` flag.

Reply:
Validation is done on save and the form will show the errors and a toastr err message will be shown so I guess it can stay enabled.